editor/src/core/shared/element-path.ts

```diff
@@ -1139,3 +1139,15 @@ export function getStoryboardPathFromPath(path: ElementPath): ElementPath | null
 export function appendTwoPaths(base: ElementPath, other: ElementPath): ElementPath {
   return elementPath([...base.parts, ...other.parts])
 }
+
+export function multipleElementPathsWithTheSameUid(paths: Array<ElementPath>): boolean {
```

Comment:
Would this be a more suitable name?
```suggestion
export function allPathsHaveTheSameUid(paths: Array<ElementPath>): boolean {
```

Reply:
First I used a similar name but I wanted the function to return `false` when there is only one uid (or when the array is empty), and for these cases `allPathsHaveTheSameUid` would be misleading a name.

Reply:
Hmmm I agree with that reasoning, but this name now gives the impression that it is checking for the presence of any duplicate UIDs, rather than checking that all paths end in the same UID. so e.g. given the input `[ 'aaa/bbb', 'aaa/ccc', 'aaa/ccc:ddd/bbb' ]` I'd expect this function to return `true` because there are two paths ending in `bbb`. So I think for that reason it would be better to name this something `multiplePathsAllWithTheSameUID`